core/shark_turbine/aot/builtins/jittable.py

```diff
@@ -214,6 +214,13 @@ def flat_wrapped_f(*args):
         if "functorch_functionalize" in self._passes:
             transformed_f = functorch_functionalize(transformed_f, *flat_pytorch_args)
 
+        for node in transformed_f.graph.nodes:  # type: ignore
```

Comment:
isn't this being done in fx_importer.py? if we need it here instead should we remove it there?

Reply:
Seems to be covered by torch-mlir's fx importer. Removing it here.